### PR TITLE
Fixed installer downloading pre-release version even when set to false

### PR DESCRIPTION
**Pull request by BoltonDev**

### CODE REVIEW

**Review by louis1706 — APPROVED** [review]

File: Exiled.Installer/Program.cs [hunk 1]
```diff
@@ -280,20 +279,27 @@ static PathResolution TryParse(string s)
...
-                    continue;
+            var enumerable = releases.ToList();
 
-                if (targetVersion.IsPreRelease && !args.PreReleases)
-                    continue;
+            foreach (var release in enumerable)
+            {
+                if (targetVersion != null)
+                {
+                    if(targetVersion == new Version(release.TagName))
```
Comment (NaoUnderscore) [inline Exiled.Installer/Program.cs:290]:
```suggestion
                    if (targetVersion == new Version(release.TagName))
```

File: Exiled.Installer/Program.cs [hunk 2]
```diff
@@ -280,20 +279,27 @@ static PathResolution TryParse(string s)
...
 
-            foreach (Release r in releases)
-            {
-                if (targetVersion != new Version(r.TagName))
-                    continue;
+            var enumerable = releases.ToList();
 
-                if (targetVersion.IsPreRelease && !args.PreReleases)
-                    continue;
+            foreach (var release in enumerable)
```
Comment (NaoUnderscore) [inline Exiled.Installer/Program.cs:286]: Do not use `var`.

**Review by NaoUnderscore — CHANGES_REQUESTED** [review]

File: Exiled.Installer/Program.cs (hunk 2 above)
Comment (BoltonDev) [reply Exiled.Installer/Program.cs]: Oh yeah mb. I think it might be a good idea to prevent their use by adding an option in .editorconfig

Comment (NaoUnderscore) [reply Exiled.Installer/Program.cs]: Done

**Review by NaoUnderscore — APPROVED** [review]

**Pull request merged**